test/create_env_cluster_topics_test.go

```diff
@@ -76,30 +57,189 @@ func TestEnvClusterTopic(t *testing.T) {
+	a := assert.New(t)
+
+	// Validate Environment ID and Name
+	log.Println("Validating Environment ID and Name...")
+	envOutput := runCLICommand(t, "confluent", "env", "describe", envID, "-o", "json")
+	var envDetails map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(envOutput), &envDetails), "Failed to parse environment details")
+	a.Equal(envID, envDetails["id"], "Environment ID mismatch")
+	a.Equal(environmentName, envDetails["name"], "Environment Name mismatch")
+	log.Printf("Validated Environment ID '%s' and Name '%s' successfully.", envID, environmentName)
+
+	// Validate Kafka Cluster ID and Basic Name
+	log.Println("Validating Kafka Cluster ID and Basic Name...")
+	clusterOutput := runCLICommand(t, "confluent", "kafka", "cluster", "describe", kafkaClusterID, "-o", "json")
+	var clusterDetails map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(clusterOutput), &clusterDetails), "Failed to parse Kafka Cluster details")
+	a.Equal(kafkaClusterID, clusterDetails["id"], "Kafka Cluster ID mismatch")
+	a.Equal(kafkaClusterBasicName, clusterDetails["name"], "Kafka Cluster Basic Name mismatch")
+	a.Equal("UP", clusterDetails["status"], "Kafka Cluster Status mismatch")
+	log.Printf("Validated Kafka Cluster ID '%s', Name '%s', and Status 'UP' successfully.", kafkaClusterID, kafkaClusterBasicName)
+
+	// Validate Kafka Topic Names
+	log.Println("Validating Kafka Topic Names...")
+	topicOutput := runCLICommand(t, "confluent", "kafka", "topic", "list", "-o", "json")
+	var topics []map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(topicOutput), &topics), "Failed to parse Kafka Topics JSON output")
+	kafkaTopicNames := []string{kafkaTopicName1, kafkaTopicName2}
+
+	// Validate each topic exists in the CLI output
+	for _, expectedTopicName := range kafkaTopicNames {
+		found := false
+		for _, topic := range topics {
+			if topic["name"] == expectedTopicName {
+				found = true
+				break
+			}
+		}
+		a.True(found, fmt.Sprintf("Expected topic '%s' not found in Kafka Topics list", expectedTopicName))
+		log.Printf("Validated Kafka Topic '%s' successfully.", expectedTopicName)
+	}
 
-		output = terraform.Output(t, runOptions, "kafka_topic_name")
-		a.Equal(output, "squad_raw_service_example_1_entity")
+	// Validate Sink Connector IDs
+	log.Println("Validating Sink Connector IDs...")
+	connectClustersOutput := runCLICommand(t, "confluent", "connect", "cluster", "list", "-o", "json")
+	var connectClusters []map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(connectClustersOutput), &connectClusters), "Failed to parse Connect Clusters JSON output")
+
+	gcsSinkFound := false
+	bigQuerySinkFound := false
+
+	for _, cluster := range connectClusters {
+		if cluster["id"] == gcsSinkConnectorID {
+			gcsSinkFound = true
+		}
+		if cluster["id"] == bigQueryConnectorID {
```

Comment:
also ineffective validation - should be input vs output, not output (from Terraform) compared to output (from Confluent).

Reply:
I have added few validation with terraform Inputs, and added bullet point in same ticket to improve it.